src/platform/ios/mod.rs

```diff
@@ -341,11 +347,16 @@ impl Window2 {
     }
 
     #[inline]
-    pub fn set_maximized(&self, maximized: bool) {
+    pub fn set_maximized(&self, _maximized: bool) {
     }
 
     #[inline]
-    pub fn set_fullscreen(&self, state: FullScreenState) {
+    pub fn set_fullscreen(&self, _monitor: Option<RootMonitorId>) {
+    }
+
+    #[inline]
+    pub fn get_current_monitor(&self) -> RootMonitorId {
+        unimplemented!()
```

Comment:
iOS provides only one dummy monitor. This function should return `RootMonitorId(MonitorId)`.